security/pkg/nodeagent/secretfetcher/secretfetcher.go

```diff
@@ -38,8 +38,6 @@ import (
 )
 
 const (
-	secretResyncPeriod = 15 * time.Second
```

Comment:
Please add an env variable, so we can set it if there are problems with 0.

Reply:
Done. Added "SECRET_WATCHER_RESYNC_PERIOD". Thanks.